src/Calculate.ts

```diff
@@ -516,7 +516,12 @@ export function calculateOfferings (
     q *= 1.05
   }
   q /= calculateSingularityDebuff('Offering')
-  q = (Math.floor(q) * 100) / 100
+
+  // This calculation is not necessary after the singularity
+  if (player.singularityCount < 1) {
+    q = (Math.floor(q) * 100) / 100
```

Comment:
This calculation is never necessary, suggest just removing it

Reply:
ok